pkgs/servers/sql/postgresql/ext/postgis.nix

```diff
@@ -24,7 +25,7 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-dOs1bj+F8UIzeRATNgiBtnSPeAgcxoj/nW8PZzp2LRM=";
   };
 
-  buildInputs = [ libxml2 postgresql geos proj gdal json_c protobufc ]
+  buildInputs = [ libxml2 postgresql geos proj gdal json_c protobufc pcre2.dev ]
```

Comment:
Hmmm, do we need the `dev` input? Wouldn't normal `pcre2` work fine? I know there are cases where it's necessary (the reasons escape me) but if `.dev` is needed, I'd believe it.

Reply:
I wasn't sure either.. had to ask in matrix :D - `.dev` just brings in the libraries, while `pcre2` would bring in the bins too. I think for this case `.dev` is correct.